diff -ru open_jtalk_original/jpcommon/jpcommon_label.c open_jtalk/jpcommon/jpcommon_label.c
--- open_jtalk_original/jpcommon/jpcommon_label.c	2011-12-20 13:25:06 +0900
+++ open_jtalk/jpcommon/jpcommon_label.c	2012-04-25 18:53:38 +0900
@@ -270,6 +270,7 @@
       if (index == a)
          break;
    }
+   if (i > 3) i = 3;
    return i;
 }
 
@@ -369,6 +370,7 @@
 
    for (i = 0, index = m->next; index != NULL; index = index->next)
       i++;
+   if (i > 10) i = 10;
    return index_mora_in_utterance(m) + i;
 }
 
